### PR TITLE
Support/wagtail42

### PR DESCRIPTION
**Pull request by nickmoreton**
This updates the package top support Wagtail>=4.1

and removes compatibility support for Wagtail versions < 4.1

Support ticket: https://torchbox.monday.com/boards/1124794299/pulses/1124794780

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 install_requires = [
     "factory-boy>=3.2",
-    "wagtail>=2.15,<5.0",
+    "wagtail>=4.1,<5.0",
```
[the remark this answers is not in the document]
Comment (nickmoreton) [reply setup.py]: This is done thanks.

**Comment by easherma** [comment]
Plenty of folks would still be using wagtail versions < 4.1. Would this make the library unusable in those cases?

**Comment by zerolab** [comment]
> Plenty of folks would still be using wagtail versions < 4.1. Would this make the library unusable in those cases?

@easherma it would make the *next* release unusable. All current versions will continue to work.

**Comment by cnk** [comment]
Does this PR still make sense given this item in the Wagtail 5.0 Release notes: 

- Include wagtail-factories in wagtail.test.utils to avoid cross-dependency issues (Matt Westcott)

**Comment by zerolab** [comment]
Superceded by #74

**Pull request closed without merging**